src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -0,0 +1,82 @@
+(ns status-im.ui.screens.discover.components.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :refer [subscribe dispatch]]
+            [clojure.string :as str]
+            [status-im.components.react :refer [view text image touchable-highlight]]
+            [status-im.ui.screens.discover.styles :as st]
+            [status-im.components.status-view.view :refer [status-view]]
+            [status-im.utils.gfycat.core :as gfycat]
+            [status-im.utils.identicon :refer [identicon]]
+            [status-im.components.chat-icon.screen :as ci]
+            [status-im.utils.platform :refer [platform-specific]]
+            [status-im.components.icons.vector-icons :as vi]
+            [status-im.i18n :as i18n]))
+
+(defn title [label-kw spacing? action-kw action-fn]
+  [view (merge st/title
+               (when spacing? {:margin-top 16}))
```

Comment:
can we move this condition to styles?

Reply:
Will do that and will check if we even still need the condition.